### DISCUSSION TITLE
Error with tiktoken, “no matching distribution found for tiktoken”

### DISCUSSION THREAD

**Discussion by byteforcebuilds** [acheong08/ChatGPT#1125] [Q&A]
Having an issue when trying to install revChatGPT, as tiktoken refuses to install. I’m on Python 3.7.3. The exact output from the shell I receive when I plug in pip3 install tiktoken is 

ERROR: Could not find a version that satisfies the requirement tiktoken (from versions: none)
ERROR: No matching distribution found for tiktoken

Any help or suggestions would be appreciated, I’m very new to all of this.

**Comment by acheong08** [comment]
Unsupported Python version

**Reply by acheong08** [reply]
Upgrade to 3.10
